roles/edpm_libvirt/molecule/default/prepare.yml

```diff
@@ -125,3 +131,61 @@
                                     'allow rwx pool=\1') | join(', ') }}
       vars:
         pools: ['vms', 'volumes', 'images']
+
+    - name: create pki temp directory
+      become: true
+      ansible.builtin.file:
+        path: "{{ edpm_libvirt_tls_cert_src_dir }}"
+        state: directory
+        owner: root
+        group: root
+        mode: 0777
+
+    - name: Create private key (RSA, 4096 bits)
+      community.crypto.openssl_privatekey:
+        path: "{{ edpm_libvirt_tls_cert_src_dir }}/ca.key"
+
+    - name: Create a certificate signing request for ca
+      community.crypto.openssl_csr_pipe:
+        privatekey_path: "{{ edpm_libvirt_tls_cert_src_dir }}/ca.key"
+        organization_name: fake-corp
+        basic_constraints:
+          - 'CA:TRUE'
+        subject_alt_name:
+          - "DNS:compute-1.example.com"
+          - "DNS:compute-1"
```

Comment:
if i have to update the hostname in the virsh line ill need to update it here too to inventory_hostname

compute-1 is the host name used in the vagrant molecule env but in our ci job we are suing the deleaged plugin and the hostname wont be static for the zuul vm